src/features/silent_walk.cpp

```diff
@@ -0,0 +1,42 @@
+#include "features.h"
+#include "../settings/globals.h"
+#include "../helpers/console.h"
+#include "../helpers/input.h"
+
+namespace silent_walk
+{
+	bool is_enabled(CUserCmd* cmd)
+	{
+		if (!g::local_player || !g::local_player->IsAlive())
+			return false;
+
+		if (!settings::misc::silent_walk)
+			return false;
+
+		auto weapon_handle = g::local_player->m_hActiveWeapon();
```

Comment:
I dont see a reason why would You want to define weapon_handle and not use it to get some data, returning false just does nothing.